package.json

```diff
@@ -4,7 +4,7 @@
   "description": "the front-end part of the authors haven application",
   "main": "index.js",
   "scripts": {
-    "test": "jest \"--coverage\"",
+    "test": "jest \"--coverage\" \"-u\"",
```

Comment:
Is the purpose of adding the `-u` flag for updating snapshots? If yes, then there should be a separate command for that actually.

Reply:
Ok, but wouldn't there be an issue with how the test runs. Since there are now two test commands